Security/Util/String.php

```diff
@@ -5,6 +5,8 @@
 /**
  * String utility functions.
  *
+ * @deprecated This class will break in PHP 7; use hash_equals() instead
```

Comment:
You should add a deprecation warning too, to warn people in their logs and the profiler (otherwise they will miss the deprecation)